p2p/src/conn.rs

```diff
@@ -244,7 +244,7 @@ fn poll<H>(
 	let _ = thread::Builder::new()
 		.name("peer".to_string())
 		.spawn(move || {
-			let sleep_time = time::Duration::from_millis(1);
+			let sleep_time = time::Duration::from_millis(5);
```

Comment:
I saw @mcdallas reported 10ms test result, are we sure the `5ms` here has the same improvement?

Reply:
It won't be the same improvement, but it will be close. Setting the conn thread sleep too high could have negative consequences, especially for miners. We can continue to tweak it as necessary, but this seems a happy middle ground for now.